code/game/objects/items/weapons/tools.dm

```diff
@@ -358,7 +358,7 @@
 	var/safety = user:eyecheck()
 	if(ishuman(user))
 		var/mob/living/carbon/human/H = user
-		var/obj/item/organ/eyes/E = H.internal_organs_by_name["eyes"]
+		var/obj/item/organ/internal/eyes/E = H.get_int_organ(/obj/item/organ/internal/eyes)

```

Comment:
One potential problem for IPCs - they were protected from this because their eyes were pathed as "optics" rather than "eyes" due to the `vision_organ` species variable, and simply making optics a child of eyes will leave them vulnerable to welding, among other things. Should a proc for eyes be implemented, called "handle too much light" or something similar? Iunno I suck at naming things right now